app/models/miq_queue.rb

```diff
@@ -174,6 +174,12 @@ def self.get(options = {})
     result
   end
 
+  # This are the queue calls related to worker management which
+  # might not be needed once we use kubernetes for worker/pod management
+  def self.put_deprecated(*args)
```

Comment:
So, I've been using a branch on my fork for all of my new architecture work.  I've only been putting things on master that are relevant to master.  I guess I don't see the value in putting this on master unless we're going to start deprecating things.